Source/drlg_l1.cpp

```diff
@@ -502,6 +502,41 @@ BYTE CornerstoneRoomPattern[27] = {
  */
 BYTE L5ConvTbl[16] = { 22, 13, 1, 13, 2, 13, 13, 13, 4, 13, 1, 13, 2, 13, 16, 13 };
 
+enum CatedralTile : uint8_t {

```

Comment:
```suggestion
enum CathedralTile : uint8_t {
```

I think this could be  a scoped enum (`enum class`) without casting issues if `L5dflags` can use `CathedralTile` as the backing type? Haven't looked into how it's used to see how difficult that'll be.